### DISCUSSION TITLE
Improve Searching the Docs on the website

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#1482] [Feature Requests]
Would it be possible to allow a search in the docs on the website for a specific version of FreeForm?
I am trying to search for implementation of 'required' but when I do a search I only get a few results all from older versions of FreeForm with ExpressEngine being the first set of links.

It's a little cumbersome to find the information at the moment. 
Freeform is vast and offers tons of customisation and so having a better way to source and find this would be a massive improvement on the documentation.  

Thanks
Terry

**Comment by kjmartens** [comment]
Thanks for your feedback @terryupton!

This is actually already underway. We hope to have a new version of the Freeform documentation (with isolated searching on versions, etc) available next week. 🙂

**Reply by terryupton** [reply]
Amazing. Thanks @kjmartens
